remote/rpc/nats/bus.go

```diff
@@ -285,6 +292,17 @@ func (r *natsPlatform) SyncStatus(ref string) ([]string, error) {
 	return response.Result, extractError(response.ErrorResponse)
 }
 
+func (r *natsPlatform) PublicSSHKey(regenerate bool) (ssh.PublicKey, error) {
+	var response PublicSSHKeyResponse
+	if err := r.conn.Request(r.instance+methodPublicSSHKey, regenerate, &response, timeout); err != nil {
+		if err == nats.ErrTimeout {
```

Comment:
The SSH regeneration can take a long time depending on the algorithm. Is a 10s timeout long enough? ( I think so, just. I've seen it take about 5s).